plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -106,6 +107,7 @@ public boolean hasComposite() {
 	public void performFinish(IProgressMonitor monitor) throws CoreException {
 		if(serverSettingsWizardPage != null) {
 			serverSettingsWizardPage.getModel().updateServer();
+			serverSettingsWizardPage.getModel().getServer().setAttribute(OpenShiftServerUtils.SERVER_START_ON_CREATION, true);
```

Comment:
I see this being made in the wizard and the page with twice the same code. Why not do it upon saving the server in the model?